.circleci/config.yml

```diff
@@ -28,9 +28,9 @@ jobs:
           name: Sample Distribution Analyses
           command: ./scripts/run_in_ci.sh bash "analyses/sample-distribution-analysis/run-sample-distribution.sh"
 
-      - run:
-          name: Sample Distribution Figure
-          command: ./scripts/run_in_ci.sh Rscript figures/scripts/fig1-sample-distribution.R
+      # - run:
+      #    name: Sample Distribution Figure
+      #    command: ./scripts/run_in_ci.sh Rscript figures/scripts/fig1-sample-distribution.R
```

Comment:
Script has been removed, so I don't see any reason to keep this in here commented out. We'll have a new Fig 1 script eventually, but it might have a totally different name!

```suggestion
```